pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -70,70 +48,40 @@ import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
 contract SomeContract {
   IPyth pyth;
 
+  /**
+   * @param pythContract The address of the Pyth contract
+   */
   constructor(address pythContract) {
+    // The IPyth interface from pyth-sdk-solidity provides the methods to interact with the Pyth contract.
+    // Instantiate it with the Pyth contract address from https://docs.pyth.network/price-feeds/contract-addresses/evm
     pyth = IPyth(pythContract);
   }
 
-  function doSomething(
-    uint someArg,
-    string memory otherArg,
-    bytes[] calldata priceUpdateData
-  ) public payable {
-    // Update the on-chain Pyth price(s)
-    uint fee = pyth.getUpdateFee(priceUpdateData);
-    pyth.updatePriceFeeds{ value: fee }(priceUpdateData);
-
-    // Read the current price
-    bytes32 priceId = 0xf9c0172ba10dfa4d19088d94f5bf61d3b54d5bd7483a322a982e1373ee8ea31b;
+  /**
+     * This method is an example of how to interact with the Pyth contract.
+     * Fetch the priceUpdate from Hermes and pass it to the Pyth contract to update the prices.
```

Comment:
I think this should probably say something to the effect of "Add a priceUpdate argument to any method on your contract that needs to read the Pyth price." before discussing how to get the price update. It's not clear at the moment that the user needs to *add* this argument to their method.

Also would be nice to put in a URL or something for a price update (maybe for the ETH/USD feed below) to demonstrate that this is easy.

Reply:
Yeah,  after reading now, I see this will make it more clear.

Reply:
URL here will make the comment the comment too long.